### ISSUE TITLE
Airlock Air Contents Wiping

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#15690]
(WEB REPORT BY: minimeatwad REMOTE: 172.93.109.202:7777)
so if you have any sort of gas different from the standard air mix and an airlock closes on that gas, the gas is replaced with the air mix and this completely wipes it from existence, to replicate this try surrounding yourself with walls as a cyborg, place a airlock and remove its safeties so you can close it on yourself (it should look like this http://i.imgur.com/w6YSZMn.png ), and then have any type of gas on the tile where the airlock is, once it closes the gas contents should be wiped, this appears to apply to any other airlock as well

**Comment by MiniMeatwad** [comment]
important: it appears to only happen if you add gases, not remove, so if there is like a plasma flood or any other flood at all, the contents will be set to the default air mix

**Comment by Exxion** [comment]
In that picture, you also have airlocks surrounding you. Does this happen if there's just one airlock?

**Comment by MiniMeatwad** [comment]
yes, and it also might happen if there is low pressure with something like plasma, but not no pressure

**Comment by FudgePucker** [comment]
Air locks replace the air when opened.
Why do you think they call them air locks? /s

Does this mean I can just make a shit ton of air locks in hallways that open and close automatically on loop to replace atmospherics?

**Comment by MiniMeatwad** [comment]
yes
